eth_tester/backends/eels/utils.py

```diff
@@ -1,12 +1,36 @@
+from importlib.metadata import (
+    PackageNotFoundError,
+    version,
+)
+
+from semantic_version import (
+    Version,
+)
+
+
 def eels_is_available():
     try:
-        import ethereum  # noqa: F401
+        import ethereum_execution  # noqa: F401
 
         return True
     except ImportError:
         return False
 
 
+def get_eels_version():
```

Comment:
I don't think we need this. I'm not a big fan of the old approach to verify the version in the code. I think we should control versioning in the dependencies and leave it be. Let's just use `eels_is_available()`.

Thoughts?

Reply:
no complaints

Reply:
We should probably drop the version check in the code for py-evm as well and control that in the versioning pins as we already do. That seems like a stale check. Can be done in this PR or another.

Reply:
sgtm 👍🏼